<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinVolumeSegmentationWidget</class>
 <widget class="QWidget" name="SkinVolumeSegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkinVolumeSegmentationWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="lowerThreshold_2">
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="upperThreshold_2">
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Z</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Val</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="deleteSeedBtn">
         <property name="text">
          <string>Delete Seed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="regionGrowingBtn">
         <property name="text">
          <string>Region Growing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="contourBtn">
         <property name="text">
          <string>Contour</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QSlider" name="lowerSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="surfaceBtn_2">
       <property name="text">
        <string>Surface</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sigmaSize">
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sigma Size</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="thresholdBtn_2">
       <property name="text">
        <string>Threshold</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="upperSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>surfaceBtn_2</sender>
   <signal>clicked()</signal>
   <receiver>SkinVolumeSegmentationWidget</receiver>
   <slot>surfaceEnhancement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdBtn_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>SkinVolumeSegmentationWidget</receiver>
   <slot>threshold(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upperSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SkinVolumeSegmentationWidget</receiver>
   <slot>changeUpperThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowerSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SkinVolumeSegmentationWidget</receiver>
   <slot>changeLowerThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteSeedBtn</sender>
   <signal>clicked()</signal>
   <receiver>SkinVolumeSegmentationWidget</receiver>
   <slot>deleteSeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regionGrowingBtn</sender>
   <signal>clicked()</signal>
   <receiver>SkinVolumeSegmentationWidget</receiver>
   <slot>regionGrowing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contourBtn</sender>
   <signal>clicked()</signal>
   <receiver>SkinVolumeSegmentationWidget</receiver>
   <slot>outerContour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>surfaceEnhancement()</slot>
  <slot>threshold(bool)</slot>
  <slot>changeLowerThreshold(int)</slot>
  <slot>changeUpperThreshold(int)</slot>
  <slot>regionGrowing()</slot>
  <slot>outerContour()</slot>
  <slot>deleteSeed()</slot>
  <slot>getCoordinates()</slot>
 </slots>
</ui>
